lib/ansible/modules/net_tools/cloudflare_dns.py

```diff
@@ -1,128 +1,159 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# (c) 2016 Michael Gruener <michael.gruener@chaosmoon.net>
+# Copyright: (c) 2016 Michael Gruener <michael.gruener@chaosmoon.net>
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
 ---
 module: cloudflare_dns
-author: "Michael Gruener (@mgruener)"
+author:
+- Michael Gruener (@mgruener)
 requirements:
-   - "python >= 2.6"
+   - python >= 2.6
 version_added: "2.1"
-short_description: manage Cloudflare DNS records
+short_description: Manage Cloudflare DNS records
 description:
    - "Manages dns records via the Cloudflare API, see the docs: U(https://api.cloudflare.com/)"
 options:
   account_api_token:
     description:
-      - >
-        Account API token. You can obtain your API key from the bottom of the Cloudflare 'My Account' page, found here: U(https://dash.cloudflare.com/)
+    - Account API token.
+    - "You can obtain your API key from the bottom of the Cloudflare 'My Account' page, found here: U(https://dash.cloudflare.com/)"
+    type: str
     required: true
   account_email:
     description:
-      - "Account email."
+    - Account email.
+    type: str
     required: true
   algorithm:
     description:
-      - Algorithm number. Required for C(type=DS) and C(type=SSHFP) when C(state=present).
+    - Algorithm number.
+    - Required for C(type=DS) and C(type=SSHFP) when C(state=present).
     type: int
-    version_added: 2.7
+    version_added: '2.7'
   cert_usage:
     description:
-      - Certificate usage number. Required for C(type=TLSA) when C(state=present).
-    choices: [ 0, 1, 2, 3 ]
+    - Certificate usage number.
+    - Required for C(type=TLSA) when C(state=present).
     type: int
-    version_added: 2.7
+    choices: [ 0, 1, 2, 3 ]
+    version_added: '2.7'
   hash_type:
     description:
-      - Hash type number. Required for C(type=DS), C(type=SSHFP) and C(type=TLSA) when C(state=present).
-    choices: [ 1, 2 ]
+    - Hash type number.
+    - Required for C(type=DS), C(type=SSHFP) and C(type=TLSA) when C(state=present).
     type: int
-    version_added: 2.7
+    choices: [ 1, 2 ]
+    version_added: '2.7'
   key_tag:
     description:
-      - DNSSEC key tag. Needed for C(type=DS) when C(state=present).
+    - DNSSEC key tag.
+    - Needed for C(type=DS) when C(state=present).
```

Comment:
Not sure if that is the purpose of this PR. Because I am not adding any new documentation I am just reformatting for better readability. Proofreading the existing module docs is a work on its own that I don't plan to take part of :-)

That said, I don't mind making these fixes if they get reported.